pkgs/tools/filesystems/hdl-dump/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  version = "2020-02-05";
```

```suggestion
  version = "unstable-2020-02-05";
```

From the [nixpkgs manual](https://nixos.org/nixpkgs/manual/#sec-package-naming):
`If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".`